Makefile

```diff
@@ -6,3 +6,22 @@ build:
 test:
 	@go test ./...
 
+.PHONY: lint
+lint: ## Run lint checks
+lint: lint-golint lint-yamllint
+
+.PHONY: lint-golint
+lint-golint:
+	@echo Running Go linter ...
+	@./hack/golangci-lint run --build-tags=e2e
+
+.PHONY: lint-yamllint
+lint-yamllint:
+	@echo Running YAML linter ...
+	## TODO(lianghao208): add other directories later
+	@./hack/yamllint changelogs/
+
+.PHONY: lint-codespell
+lint-codespell: CODESPELL_SKIP := $(shell cat .codespell.skip | tr \\n ',')
```

Comment:
is `codespell` better than [misspell](https://github.com/golangci/misspell) thats part of `golangci-lint` ?
also if we do end up using `codespell`, the target needs to be added to `lint` :)

Reply:
cc @skriss

Reply:
Since projectcontour/contour is using codespell as well, maybe we should refer to contour's approaching? 

Already added codespell to lint, thanks for pointing that out.

Reply:
Codespell will check comments as well.